rest/session_api.go

```diff
@@ -243,6 +243,7 @@ func (h *handler) getUserSession() error {
 // that the session being deleted is associated with the user.
 func (h *handler) deleteUserSession() error {
 	h.assertAdminOnly()
+	h.setHeader("Content-Type", "application/json")
```

Comment:
Same here - no response body, so we shouldn't set a content type.

Reply:
Fixed in latest revision.